### DISCUSSION TITLE
Preserve cursor when dragging QSplitter

### DISCUSSION THREAD

**Discussion by zeodtr** [quasarframework/quasar#14845] [General - Components / Directives / etc]
QSplitter changes the cursor to 'grabbing' when dragging the separator. I want to disable that behavior.

How can it be done?

BTW, I've looked at the source code of QSplitter and found that it uses TouchPan, and TouchPan is the one actually changes the cursor. But TouchPan has 'preserveCursor' attribute used to preserve the cursor. If QSplitter allows that attribute via prop setting, The cursor can be preserved.

Thanks.

**Comment by MilosPaunovic** [comment]
Use `separator-class` or `separator-style` to achieve that behaviour.
https://codepen.io/milospaunovic/pen/JjZMPzB?editors=1010

**Reply by zeodtr** [reply]
Your codepen still shows the grabbing cursor when dragging the separator.

**Comment by metalsadman** [comment]
use css
```css
html {
  cursor: default !important;
}
```
or do it on your q-splitter's `update:model-value` event + `emit-immediately` prop changing the html style `document.documentElement.style.cursor = ''`, has slight flicker tho as it updates the cursor.
